src/meta/src/barrier/checkpoint/creating_job/mod.rs

```diff
@@ -140,6 +139,11 @@ impl CreatingStreamingJobControl {
             .flat_map(build_actor_connector_splits)
             .collect();
 
+        assert!(
```

Comment:
Will this line of code be executed in this scenario?
SET streaming_use_snapshot_backfill to TRUE;
CREATE TABLE cdc_table (...) FROM cdc_source TABLE '...';

Reply:
No, because snapshot backfill only works for backfilling from RW tables/MVs.